### ISSUE TITLE
[iOS] Printing to "(null)" pop up

### ISSUE THREAD

**Issue by dsjdigital** [katzer/cordova-plugin-printer#164]
Is there a way to hide this (native) pop up while sending the print job?
Or... if not, perhaps there is a way to replace "(null)" by another text that makes more sense?

**Comment by Wolrab1** [comment]
Did you get anywhere with this one @dsjproductions ? having the same issue. Blows my mind how this plugin is recommended on Ionic Native with no development.

**Comment by dsjproductions** [comment]
Nope, unfortunately not. We eventually decided to print entirely through the back end using Google Cloud Printer... This cordova/ionic plugin should not be recommended, indeed!

**Comment by dsjdigital** [comment]
> **Printing to "(null)"** means that you specified an invalid `printerId`. The system tries to connect to the printer but does not find it.

Well... printing actually works fine even though this popup is being shown.

**Comment by katzer** [comment]
@dsjdigital Do you use Ionic or plain Cordova project?

**Comment by dsjdigital** [comment]
> @dsjdigital Do you use Ionic or plain Cordova project?

Plain Cordova

**Comment by katzer** [comment]
And you're sure that you haven't assigned a value to `printerId`?
